osquery/tables/system/linux/processes.cpp

```diff
@@ -56,6 +56,26 @@ inline std::string readProcLink(const std::string& attr, const std::string& pid)
   return result;
 }
 
+Status getPidsForContext(QueryContext& context, std::set<std::string>& pids) {
+  auto s = osquery::procProcesses(pids);
```

Comment:
I'm interested in the latency incurred by visiting all leafs in the `/proc` directory.